f/fcs/fn_fcsPersistentEffects.sqf

```diff
@@ -0,0 +1,52 @@
+// F3 - FCS/Commander's Override system
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// Originally from: https://github.com/NikkoJT/njt_fcs (integrated with permission)
+
+// This component handles the FCS failure symptoms, such as the HUD warning and zeroing overrides. It's launched when an FCS failure is determined by the Hit EH added in fcs_init.
+
+params ["_target"];
+
+// It's all a big loop
+while {(_target getVariable ["fcs_failure",false]) && (alive _target)} do {
+
+	// Check whether there is a gunner
+	if !(isNull (gunner _target)) then {
+		_currentGunner = gunner _target;
+		{
+			// Get the muzzles for every weapon on the main turret and force set their zeroing
+			private _targetWeapon = _x;
+			private _targetWeaponMuzzles = getArray (configFile >> "CfgWeapons" >> _x >> "muzzles");
+			{
+				_targetMuzzle = _targetWeapon;
+				if !(_x == "this") then {
+					_targetMuzzle = _x;
+				};
+				_zeroCheck = _currentGunner setWeaponZeroing [_targetWeapon,_targetMuzzle,1];
```

Comment:
Is `setWeaponZeroing` Global-Argument,Global-Effect?

Reply:
The wiki helpfully does not say. I think I remember testing it and having it work as expected on remote units, but I'll have to check again to prove it.